### ISSUE TITLE
Acknowledge the limitations of available parliamentary voting data in the Divisions Overview

### ISSUE THREAD

**Issue by katska** [openaustralia/theyvoteforyou#1326]
Acknowledge the limitations of available voting data in the Divisions Overview on the [Divisions](https://theyvoteforyou.org.au/divisions/all) page, change

_In the Australian Parliament avote is called a division. Through these votes, elected members shape legislation that affects us all. Learn more..._
to 
_Parliament in Australia, formal votes are known as divisions. This is named after the traditional practise of physically moving in the room to divide into those who say Aye (yes) or no response to a question. Amazingly, they still get up and walk around to vote even now (why?) Through these votes, elected members shape legislation that affects us all. Learn more..._

there's lots more - its a start...

**Comment by mlandauer** [comment]
I think the intent of this issue is an important one. I don't think the suggested new wording quite gets it right because it focuses on what happens during divisions rather than explaining that most of the votes are missing from the record because they're on the "voices".

**Comment by katska** [comment]
Divisions are called to understand the majority decision (or will of the house)
The side effect is a lack of accountability

Specifically,
1) It doesn't matter how important the vote is, that's unrelated to whether a division is called. Division may be called for a trivial matter, and not called on a decision that affects all Australians. 

2) if only divisions are recorded for all Representatives, no-one has access to a full voting record for their Representative on which to call them to account.

3) Unless enough independents disagree with the majority to call a division, their dissent is not recorded in a way that we are able to meaningfully present currently. *

Point 3 is referred to in the help/faq already, but not the fact that this is a different kind of record than the division votes, and the other points are not mentioned in this section. 
@mackaymackay  @mlandauer Does this cover off the question 'why should people care that not all votes are recorded'? which is really the question I'm answering here.

**Comment by mlandauer** [comment]
@katska I don't really understand what you're saying here because I though we were discussing changing a specific bit of wording around the divisions which really only has room for maybe one sentence. So, inherently there's a whole bunch of nuance that we can't include in those one bit here. That would need to go elsewhere.

**Comment by katska** [comment]
this was a discussion of the issues underneath that. To make a short sentence that reflects a more complex truth, we also need to understand how we talk about he more nuanced reality. That was we're clear on what we're trying to reflect in the short snappy version.

For the specific wording @mlandauer how about we make sure to include those words in the high level overview of divisions.
At the moment this reads:

Divisions 2022
"In the Australian Parliament a vote is called a division. Through these votes, elected members shape legislation that affects us all. [Learn more...](https://theyvoteforyou.org.au/help/faq#division)

Which we could change to read 

Votes 2022
"TheyVoteForYou tracks divisions, the formal votes that are recorded in Parliament, to learn how our elected members vote on legislation that affects us all. Many votes go unrecorded, so we can't include them here. [Learn more...]
@mackaymackay  @mlandauer does that work for you?

**Comment by mackaymackay** [comment]
@Kat could the words "in Parliament" fit in the second sentence, or would that push it over?

> "TheyVoteForYou tracks divisions, the formal votes that are recorded in Parliament, to learn how our elected members vote on legislation that affects us all. Many votes *in Parliament* go unrecorded, so we can't include them here. [Learn more...]

**Comment by mackaymackay** [comment]
Then again, those other votes don't go unrecorded, they are just insufficiently recorded... Though I think trying to clarify *that* would get far too wordy, so probably best to stick with "unrecorded" even if it makes the pedant in me cringe.

**Comment by mlandauer** [comment]
@mackaymackay I've taken your addition and changed it on the site. You can now see the updated text at https://theyvoteforyou.org.au/divisions/all

I'm not closing this issue yet because I think there's a few too many words. I think we could make this shorter.

**Comment by katska** [comment]
> Then again, those other votes don't go unrecorded, they are just insufficiently recorded... Though I think trying to clarify _that_ would get far too wordy, so probably best to stick with "unrecorded" even if it makes the pedant in me cringe.

@mackaymackay Given that there is no record of who is in the room and would have been available to vote yay or nay, then I believe 'unrecorded' is a reasonable assessment 'largely unrecorded' would be more accurate. I agree it is unsatisfying not to be able to account for those who are left out in their opposition to a particular decision because it never made it to division. 

Also I think the Parliamentary Education Office information doesn't paint a full picture of the situation based on what we've been told recently

https://peo.gov.au/understand-our-parliament/your-questions-on-notice/questions/can-you-please-explain-voting-on-the-voices-and-the-division-process

@mackaymackay If you can think of a way we can characterise the minority but important cases where on the voices votes have dissent registered or if there's a way to discuss those impartially then I think that's also a worthwhile piece of writing.


the only way I can think to put this at the moment looks like this, but its not a readable overview.

Elected representatives in Australian Parliament make decisions by voting 'on the voices', verbal votes that go largely unrecorded and by voting in divisions which are recorded. These are the votes we can hold them to account with.
[Learn more...](https://theyvoteforyou.org.au/help/faq#division)

Shorter version could be 

Votes 2022 
In the Australian Parliament, elected members vote  in divisions and 'on the voices' on legislation that affects us all. [Learn more...]

**Comment by mlandauer** [comment]
> In the Australian Parliament, elected members vote in divisions and 'on the voices' on legislation that affects us all. [Learn more...]

I think introducing two technical terms "divisions" and "on the voices" in a bit of text that is supposed to be explaining what a "division" is feels a little circular and technical. It might be technically correct but doesn't explain what "on the voices" is, why they are different than divisions and why divisions are included but on the voices aren't.

This is a really difficult one...
